README.md

```diff
@@ -36,13 +36,13 @@ $ yarn add @fauna-labs/serverless-fauna
 
 > NOTE: This package has not reached a 1.0 release yet. Minor version releases may still contain breaking changes. If you wish, you can restrict your projects to only accepting patch updates by prefacing the version number with a `"~"` in your `package.json` file. For example, `"~0.2.0"` or `"~0.1.6"`
 
-## FQL X (beta)
+## FQL 10 (beta)
```

Comment:
we need a consistent naming scheme here. in the database we have this ticket, to fix the naming: https://faunadb.atlassian.net/browse/ENG-4996, which is unclear, so I'm not sure what the right name is (FQL v10 vs FQL 10). I think it makes sense to double check the names with product, and its a good excuse to get a consistent set of naming rules in place.

Reply:
Actually, I'll go ahead and revert the readme for now